docs/man/sesman.ini.5.in

```diff
@@ -117,8 +117,7 @@ defaults to \fI10\fR.
 
 .TP
 \fBMaxSessions\fR=\fInumber\fR
-Sets the maximum number of simultaneous sessions. If not set or set to
-\fI0\fR, unlimited session are allowed.
+If set to \fI0positive value\fR, limits the maximum number of concurrent sessions.
```

Comment:
The original text doesn't mention how it behaves if a negative value is set. However, it's not a real issue. Let's get back to the original text.